### PR TITLE
chore: update storagemanager docs

### PR DESCRIPTION
**Pull request by dbanksdesign**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx [hunk 1]
```diff
@@ -9,8 +9,8 @@ FileUploaderComponentsExample, LevelExample, VariationExample, AutoUploadExample
 MaxFileSizeExample, ResumableExample, ShowImagesExample, FileUploaderSuccessExample, PreviewerComponentsExample, TrackerComponentsExample, UploadButtonComponentsExample } from './examples'
 import ReactPropsTable from '@/components/propsTable/ReactPropsTable';
 
-<Alert variation="info" heading="Developer Preview">
-  The File Uploader component is in developer preview! 
+<Alert variation="warning" heading="Developer Preview">
+  FileUploader has exited Dev Preview and was renamed to [StorageManager](storagemanager) with some API changes. Please migrate to the StorageManager component, as the FileUploader component will be removed in a future major release.
```
Comment (ioanabrooks) [inline docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx:13]: Should the link be `[StorageManager](/storagemanager)`?

File: docs/src/pages/[platform]/connected-components/storage/storagemanager/react.mdx [hunk 2]
```diff
@@ -35,19 +43,50 @@ To use the StorageManager component import it into your React application with t
...
+    ```
+  </ExampleCode>
+</Example>
+
+
+## Pausable / Resumable Uploads
 
-## Modify file and name
+A resumable upload will upload the file in chunks. This allows users to pause an upload and resume it at a later time. You will typically want to do this only when the expected files are larger than the chunk size, which is 5mb.
+
```
Comment (ioanabrooks) [inline docs/src/pages/[platform]/connected-components/storage/storagemanager/react.mdx:74]: `5mb` -> '5MB`

File: docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx (hunk 1 above)
Comment (dbanksdesign) [reply docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx:13]: I think the starting slash would mean that link would go to [domain]/storagemanager, without the slash it is relative to the current path so [domain]/react/connected-components/fileuploader -> [domain]/react/connected-components/storagemanager At least that is my understanding

File: docs/src/pages/[platform]/connected-components/storage/storagemanager/examples/StorageManagerEventExample.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+          // remove the file from the list
+          filesRef.current = filesRef.current.filter(
+            (file) => file.key !== key
+          );
+          setFiles(filesRef.current);
+        }}
+        onUploadSuccess={({ key }) => {
+          filesRef.current = [...filesRef.current, { key }];
+          setFiles(filesRef.current);
+        }}
```
Comment (calebpollman) [inline docs/src/pages/[platform]/connected-components/storage/storagemanager/examples/StorageManagerEventExample.tsx:27]: Not fully grasping why we need the ref here, can these just be:
```suggestion
        onFileRemove={({ key }) => {
          setFiles((prevFiles) => prevFiles.filter(
            (file) => file.key !== key
          ));
        }}
        onUploadSuccess={({ key }) => {
          setFiles((prevFiles) => [...prevFiles, { key });
        }}
```

**Pull request merged**